app/models/order_item.rb

```diff
@@ -1,4 +1,12 @@
 class OrderItem < ApplicationRecord
+  after_create :set_price
   belongs_to :item
   belongs_to :order
+  
+  validates :quantity, presence: :true
+
+  def set_price
```

Comment:
This might not be the best way to do this but I didn't want to pass price along with everything else from the front end if I already had an easy way of grabbing it